fdbserver/Knobs.h

```diff
@@ -455,6 +455,7 @@ class ServerKnobs : public Knobs {
 	bool REPORT_DD_METRICS;
 	double DD_METRICS_REPORT_INTERVAL;
 	double FETCH_KEYS_TOO_LONG_TIME_CRITERIA;
+	double MAX_STORAGE_COMMIT_TIME;
```

Comment:
Please add a comment, something like:
The max fsync stall time before mark a disk as failed.